src/styles/theme/_variables.scss

```diff
@@ -11,7 +11,11 @@ $color-surface: #FFFFFF;
 $color-on-surface: #161616;
 $color-tertiary: #B34000;
 $color-cej-background : #EA711B;
-
+$color-homepage-first-section: #E8EDFF;
+$color-homepage-second-section: #FEF3FD;
+$color-homepage-third-section: #F1FEF5;
+$color-homepage-fourth-section: #FFF4ED;
```

Comment:
J'aurai fait autrement (évidemment tu me diras) :
* variabiliser les couleurs des icons, que j'aurai nommé explicitement par la couleur la plus proche (utilise https://chir.ag/projects/name-that-color/)
* utiliser ces variables via le css `color`, via une className précisée sur le svg
* utiliser le lighten de sass à partir de ces couleurs pour définir le background color. Attention la valeur du lighten diffèrera sûrement d'une section à l'autre et tu ne tombera pas exactement sur la même couleur, pas bien grave

Qu'en penses-tu ?